### DISCUSSION TITLE
换了新电脑，求个激活码，谢谢

### DISCUSSION THREAD

**Discussion by pppapy** [LFTEC/LearningFucker#50] [General]
EukRx1EMsioFXLMwEKSCbkALfnpGS21HEud4EgfyFTMfDBWx+G/M1zWstmwXoPZ7Htq
Bb6+WY8c1z/km9TVuQfnBTv3cMuunr9BpySE4p6ch1JgESvYZ2iguyKpMqOVnCCHPOp
lY6E5YVCnrtaxbAB2ojy/JvbrDMwZL92+xthk4LvRu0YnaqTUCScPXXAsZ9ssHBWaDv
FlnQRMvzHEcA3VoitBeXZypFdGFwWJztx5kBYvXV8FEDjJR6X/8gp6ggNiTFONWM/AE
GsyRTS60Z/VSpu+tuW0QnvC237byOax39z8Otp3Rt2JaVceB2oOatyyclCZ1f0Vt8I+
hrfVxjoAJ3bfhK30eElpH77ONW95WyFoLDP7lIO2TPvckIH/qbmrtX1IWHTJTWQSebn
0KhjQwNMOLyG9SzJLGvCNPnh8UgUvxyBje6wQgQY6c9RaCL8AwBKf5DNlnlRYbDx2BI
YG2aW9rK8U3+Iccrph+DBRiUyAn2kvwzZiioUEZ8RUxTNn9lMxOFjDUzLmHuPiHprK8
J1z1WVqmJkhGCrQl9oetaw82a3eXC7O3O+lt54JqK7JXd52LrKoTcdkPOvgWou1Arb7
OKtsHjRKuLUHrfNZ7hnXfbH2IsdJsDKp4O0dzKZVO8ex/0erVWkazKVYhNiU+fNM/O/
DUiKo0S4B2/FCoMGIkv/hekdVsdEFM/js0SqWFfnA3ogwtbvVf/kCfzxPfJ1OiKbPrV
hhAOST9AdpFjzuo19XiHWcKiEztZpD6iC6HCvSABNTUAn5JyyiOzNdSElloMGC7zFdt
x3MvRrZiSCpsiXYzNgemEZ0a83jXc13weQt0ADfgTEXAScLmltcT05MzlyjhE8XYyGZ
t+UQRvjWVG3uZzj1ziF7i48jayLOctmE6uA4FQRya1yS88R4VWNw315fVHIsMrfMgDM
cma5d65n3aldBbRKGjQnuqxzc/O3dmv1COo9QyU9ldZR5Mc/UMtRweTC0S4c2NOueNQ
ASF+84WBkcs1XkXu54yakFBOfcSzhBrZKzmtLZWQV7CKenB2IpndfVdTlh6UqYazwqp
MTntp2n04kdFFKqxrNzhje5LCNR+E6tGhSY/zCGcQvRb5MPCQ9YG1RSQjuGu6Idql9t
DnLVlExMa0ikiX7QSfNqYImvSZqkQwChq450GdAUiqD1DpzNdZaDBoZjspgioG/WPCJ
w9yDspfNYzDvCCnwAfQAkHrmP1Hy78xznX2OKktC1jQMbrFoCupleCE6MpAKzwMnjnh
c4hgXp8F6Su/boofJn2YCR+NQOr6/WAs8XYQUnPopUzeAcjf5uZMjoad+2N0GoMT2E8
s2H7/Z63oNdSw5nlt5D3NJZI9w==

**Comment by LFTEC** [comment]
HiV6qi1qa/Y6XxKAuSyaOLO2cx5z256pKACzvRVUdaOzwm/gsmok0G0QutOB3NPj5+8PzV8AYMwDIUMG8GwItoNCgHcrsR4eVI0EvjfOT5Tlv2c/bjqxa0YW7sApxo75vEmITRmgDsrIyFGnC+ozYUmkgSeN1TaVqKUiX0hSjht4RrbmEIta+x+rOuaB7M/axiOITSZtGZSjUJrmklZ8Cr+PNxylXGi2fzNNMSRfKJnphS9vUyKalg8LYNm2jekvcrDN2NHaZECBG4fVqpfXcPIB1d0MG2Cf3FIy4cGeNadxKL/IDnzA7lQONDPdD9dG+vpi7Q/Fvc7yKnUhRPcmaQbjt2snC+wW/3XinncphWMLVf3Fftcsn7e+X3bDQxH9ENUZQpy1N72lNV32+9VudKFj2jR3YsiG9H6xLVQjeRRyPTpbrda4ruP1uGEk+njaL2tE3tW59OH8PajgCjRpbufDux7tbKowev17x2m7I2mPvSpoUss+kiWLNkwzoHVLyZ6Tf6vV7KhcigTYXxqtiAw0NRZpUsf4bGQSc+ZGDAF93ev/28tjyK4YZE+tvNh6KqfV/bWN1iD6R0OBt5EKc8WT75F0HxwSsMz8kNTWEF9VbIJfY5QJFO0k5rlPF+r0rs8tE+xbWC/Vjo56zKFfy+HnnW0ty6BWKtO1CN2Js4h+VtEP0n/LH1Eh7pwLpNjmflpSajHsAwXsWxmzBOrRFRZDvV94Eo9DyEv1jQ8G5ula2dsvBk0ePrOIGHVogjaaXVbn1jmLbYer7T920rHu6IL6DdMrcXRHF76kHqqVnziUu5l0c9t55R+R+ygfJaXNbkIN8LqhbQm+LLNQ3ilIDgpa/7FAw+eSKUiW1njE8GzkRVZxCg7RXrIHxa5+Sx/6m+9Pw52Vmrk1HOTt5QzC6q+sM/AMC+GMXfOjbZsQERB5iJ773RxGNrYG3vq7dBBlV7cr3cuGNKSpeoGc5US5KH/NaqWZPOo4zcNdMz+DCM2IGEUqWSFqLHNItcMHK0tZBibLFPOiswLh0kF1swT5OnRBin+fQOVr1wM03xLbor/9mQYyvkGZ+B3f1gCXmjENGgo0XrknyncsCNcHDqhfhYEwtzYTPNdH7vC9NgYeQokyid5ofiwwaZiv5z1d2gId7DXbqy2L7OUBiDNefDGdiIO5RpGtTgBw+CkYJBklNIHBznFIaS40Z8BbqMeg4nzy9YjpUtwc0vHSx89qJRO0Heo7U9vgFP6WoUc+KonmU+p1qg27bgGloqyFv+lx/mx75dJeqoOcuuCWKgE9EbGLcwiT0m5eO61Z8xQnTxE3Blnf3gJIWAgdkzxCGzjAGiQZIeww0Ig4fC+yB5fDbIG0LA==
